Src/Base/AMReX_RealBox.H

```diff
@@ -28,6 +28,11 @@ public:
     RealBox (const Real* a_lo, const Real* a_hi) noexcept
         : xlo{AMREX_D_DECL(a_lo[0],a_lo[1],a_lo[2])}, xhi{AMREX_D_DECL(a_hi[0],a_hi[1],a_hi[2])} {}
 
+    //! Construct region given diagonal points.
+    AMREX_GPU_HOST_DEVICE
+    RealBox (const Vector<Real>& a_lo, const Vector<Real>& a_hi) noexcept
```

Comment:
This cannot be host and device function because it takes Vector.

Reply:
Argh, I mean to write
```suggestion
    RealBox (const RealVect& a_lo, const RealVect& a_hi) noexcept
```